packages/headless/src/types.ts

```diff
@@ -99,6 +99,28 @@ export interface PageInfo {
   startCursor: string;
 }
 
+/**
+ * A WordPress featured image object obtained through a WPGraphQL request
+ *
+ * @export
+ * @interface FeaturedImageNode
+ */
+export interface FeaturedImageNode {
+  id: string;
+  altText: string;
+  sourceUrl: string;
+}
+
+/**
+ * A WordPress featured image node object obtained through a WPGraphQL request
+ *
+ * @export
+ * @interface FeaturedImage
+ */
+export interface FeaturedImage {
+  node: FeaturedImageNode;
+}
+
```

Comment:
Thanks for adding these. Just a heads-up, I'm going to see if we can automatically generate these from the GraphQL schema.

Reply:
That would be excellent!